### DISCUSSION TITLE
/Dock and /Undock actions  are never complete

### DISCUSSION THREAD

**Discussion by omerdurmus61** [iRobotEducation/create3_docs#489] [Questions]
### How are you connecting to your Create 3?

Wi-Fi (ROS 2)

### Computer(s) Model(s) and Operating System(s)

Ubuntu 22.04

### Which version of ROS 2 is installed on your computer?

Humble

### Which firmware version is installed on your robot?

H.2.3

### Which RMW is your robot running?

FastRTPS

### Is the robot connected to a network? If so, what is the network type?

Home network

### Are there multiple Create 3 robots connected to your network?

No

### Is multicast enabled?

None

### What is the Adapter Board's USB/BLE Toggle currently switched to?

USB

### Describe your question.

I connected to Create3 with my PC over Wi-Fi and Intel NUC over a USB port. I am using Ubuntu 22.04, ROS Humble, and FastRTPS on both computers. When I call an action using these commands

ros2 action send_goal /dock irobot_create_msgs/action/Dock {}\ 
ros2 action send_goal /undock irobot_create_msgs/action/Undock {}\ 

Dock and undock actions work perfectly but in terminal Dock action does not return a result. Although the dock action works, I am not getting any output in the terminal.

**Comment by slowrunner** [comment]
Why the back-slash char at the end?  Try Issue single call and without the backslash

**Reply by omerdurmus61** [reply]
didn't work
